Source/WebKit/UIProcess/API/glib/WebKitWebView.cpp

```diff
@@ -2501,6 +2534,22 @@ void webkitWebViewRunAsModal(WebKitWebView* webView)
 #endif
 }
 
+bool webkitWebViewHasMainResource(WebKitWebView* webView)
+{
+    ASSERT(webView != nullptr);
+    WebKitWebViewPrivate* priv = webView->priv;
+    return (priv->mainResource != nullptr);
```

Comment:
No need for the parens. Also no need to declare `priv` if it's only going to be used on one line:

```suggestion
    return webView->priv->mainResource != nullptr;
```